samplers/parser.go

```diff
@@ -331,25 +332,12 @@ func ParseMetric(packet []byte) (*UDPMetric, error) {
 	return ret, nil
 }
 
-// UDPEvent represents the structure of datadog's undocumented /intake endpoint
-type UDPEvent struct {
-	Title       string   `json:"msg_title"`
-	Text        string   `json:"msg_text"`
-	Timestamp   int64    `json:"timestamp,omitempty"` // represented as a unix epoch
-	Hostname    string   `json:"host,omitempty"`
-	Aggregation string   `json:"aggregation_key,omitempty"`
-	Priority    string   `json:"priority,omitempty"`
-	Source      string   `json:"source_type_name,omitempty"`
-	AlertLevel  string   `json:"alert_type,omitempty"`
-	Tags        []string `json:"tags,omitempty"`
-}
-
 // ParseEvent parses a packet that represents a UDPEvent.
```

Comment:
that docstring will need adjustment now (:

I also think it should probably write about what the fields on the Sample returned mean & mention the dogstatsd tags (: